SimulationAgent/SimulationRunner.cs

```diff
@@ -28,6 +28,9 @@ public class SimulationRunner : ISimulationRunner
         // ID prefix of the simulated devices, used with Azure IoT Hub
         private const string DEVICE_ID_PREFIX = "Simulated.";
 
+        // ID used for custom device models, where the list of sensors is provided by the user
+        private const string CUSTOM_DEVICE_MODEL_ID = "custom";
```

Comment:
Should this also be capitalized?

Reply:
the code assume it's a lowercase value, see the where the value is used. The value sent from the UI is lowercased, to make it case insensitive.